specification/maps/data-plane/Weather/preview/1.0/weather.json

```diff
@@ -565,13 +989,13 @@
           "description": "Type of unit for the speed value."
         },
         "unitType": {
-          "type": "integer",
-          "format": "int32",
+          "$ref": "#/definitions/UnitType",
           "description": "Numeric ID value associated with the type of unit being displayed. Can be used for unit translation. Please refer to [Weather Service Concepts](https://aka.ms/AzureMapsWeatherConcepts) for details."
```

Comment:
This `description` can be deleted since it is also in the `UnitType` definition.